release/ray_release/command_runner/anyscale_job_runner.py

```diff
@@ -222,13 +222,12 @@ def run_command(
         )
 
         full_env = self.get_full_command_env(env)
-        env_str = _get_env_str(full_env)
 
         no_raise_on_timeout_str = (
             " --test-no-raise-on-timeout" if not raise_on_timeout else ""
         )
         full_command = (
-            f"{env_str}python anyscale_job_wrapper.py '{command}' "
```

Comment:
we don't need to append the list of env_var anymore as they are handled through anyscale